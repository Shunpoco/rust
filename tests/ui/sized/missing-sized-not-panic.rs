// ICE-141806
trait Trait<T> {}
fn func(x: *const dyn Trait<()>)
where
    Missing: Sized, //~ ERROR E0412
{
    let _x: *const dyn Trait<u8> = x as _;
}

fn main() {}
