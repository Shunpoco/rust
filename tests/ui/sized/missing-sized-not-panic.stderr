error[E0412]: cannot find type `Missing` in this scope
  --> $DIR/missing-sized-not-panic.rs:5:5
   |
LL |     Missing: Sized,
   |     ^^^^^^^ not found in this scope

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0412`.
